src/main/scala/ai/starlake/schema/model/Metadata.scala

```diff
@@ -228,6 +247,57 @@ case class Metadata(
     )
   }
 
+  /** Keep metadata that are different only
+    *
+    * @param parent
+    *   : Schema level metadata
+    * @return
+    *   the metadata that differs between parent and this element.
+    */
```

Comment:
Move this code to Metadata

Reply:
It's already in Metadata, do you think we should move this to the object Metadata ?

Reply:
My bad